apps/src/applab/applab.js

```diff
@@ -666,6 +666,9 @@ Applab.init = function(config) {
   studioApp.runButtonClick = this.runButtonClick.bind(this);
 
   Applab.channelId = config.channel;
+  if (config.level.testAssetPathPrefix) {
+    Applab.assetPathPrefix = config.level.testAssetPathPrefix;
+  }
```

Comment:
Just specifying Applab.assetPathPrefix in runBeforeClick in the test definition doesn't work, because it gets called after Applab.init and so the asset path prefixes have already been added. I could add a call to a new method options.onBeforeInitialize in appMain.js and set that to testData.onBeforeInitialize, if a more general solution sounds better.